js/app/modules/selection/featuredFirst.js

```diff
@@ -21,13 +30,11 @@ const AllHighlights = new Module.Class({
         if (query_index == 0) {
             return Eknc.QueryObject.new_from_props({
                 limit: limit,
-                tags_match_all: ['EknArticleObject', 'EknFeaturedTag'],
-                excluded_tags: [],
+                tags_match_all: ['EknFeaturedTag'],
```

Comment:
Isn't this querying for only Featured content? So is not really "first", but "only".